languages/elixir/exercises/concept/maps/.docs/hints.md

```diff
@@ -0,0 +1,60 @@
+### General
+
+- A [map][maps] are a associative data structure of key-value pairs.
+- Elixir offers [many useful Map module functions in the standard library][map-module].
+
+### 1. Define a new high score map
+
+- You need to define a [named function][named-function] with 0 parameters.
+- It should return an empty [map][maps].
+
+### 2. Add players to the high score map
+
+- You need to define a [named function][named-function] with 3 parameters.
```

Comment:
You could consider not having these "You need to define a function with X parameters" and their "The first argument is a Y" comments as these things are also explained in the instructions. For the basics exercise it definitely made sense to have these hints as students were defining functions for the first time. 

That said, I don't mind these hints, so feel free to leave them in if you think they're useful.

Reply:
I'm not sure if they would be helpful or not, they seem redundant for me, but I am not looking at the language with fresh eyes.

If I were to remove them, is there a better hint to leave?

Reply:
> I'm not sure if they would be helpful or not, they seem redundant for me, but I am not looking at the language with fresh eyes.

I agree with this. They made sense for the very first basics exercise, but less so for subsequent exercises.

> If I were to remove them, is there a better hint to leave?

Well first of all, you don't _have_ to have a hint for each task, although one can usually find something to link to or say.  Looking at the example implementation, maybe you could link to the `Map.put` documentation (without mentioning its name) or another resource that explains how to add things to a Map?

Reply:
Sounds good, I think I have something I like more now to submit for review.